util/stress_regression_check.py

```diff
@@ -1,13 +1,15 @@
 import os
 import requests
 import datetime
+import re
 from openai import OpenAI
 from github import Github
 
 github_token = os.getenv("GITHUB_TOKEN")
 repo_name = "sustainable-computing-io/kepler-metal-ci"
 
 today = datetime.datetime.now().strftime('%Y-%m-%d')
+lastIssueDate = "2024-07-31"
```

Comment:
Will the issue date remain constant? Wouldn't this change

Reply:
as a long term goal, I am not sure if we can ref https://github.com/sustainable-computing-io/kepler-metal-ci/pull/265/files#diff-0028b6356c8200f2409c83d6d3a1a35bd5025c9aed8872595981bf0db3d7abcaR8 to make it as an environment value?
for short term, just hard code to reduce duplicate issue open as https://github.com/sustainable-computing-io/kepler-metal-ci/issues/267